libqtile/core/manager.py

```diff
@@ -1412,6 +1412,17 @@ def prev_screen(self) -> None:
         self.focus_screen((self.screens.index(self.current_screen) - 1) % len(self.screens))
 
     @expose_command()
+    def swap_screens(self, reverse = False):
+        """Switch groups between active screens"""
+        prev_group = None
+        screens = self.screens[::-1] if reverse else self.screens
+        for screen in screens:
+            if prev_group is None:
+                prev_group = screen.group
+                continue
+            screen.set_group(prev_group)
+            self.cmd_next_screen()
```

Comment:
```suggestion
            self.next_screen()
```
The `cmd_` prefix is deprecated.

Also, this changes the actual screen that's focused. Do you need to do that?